docs/tutorials/task.csv-to-oscal-cd/csv-to-oscal-cd.md

```diff
@@ -0,0 +1,151 @@
+# task csv-to-oscal-cd
+
+The below table represents the expectations of trestle task `csv-to-oscal-cd` for the contents of the input csv-file for synthesis of the output OSCAL Component Definition json-file.
+
+`Column Name` is the name of the expected column in the input csv-file. Any additional columns not identified here, for example foobar, are also extracted and placed into the output json-file as component.control-implementation.prop\["foobar"\].
+
```

Comment:
I believe the all the rules and their related props are now captured at the component level. So the text "json-file as component.control-implementation.prop\["foobar"\]." needs to be updated above and in the table below.

Reply:
Not yet. This is coming with Issue #1233 with, perhaps, other changes as well.